advanced_html_css/responsive_design/introduction_to_responsive_design.md

```diff
@@ -8,26 +9,31 @@ This section contains a general overview of topics that you will learn in this l
 - You'll learn what "Responsive Design" is and why it's important.
 
 ### Terminology
-"Responsive Design" is the term used to describe creating websites that _respond_ to changes in browser size in order to create something that works on any device. Unfortunately, that phrase is a little bit misleading, due to the word 'design'. These lessons are not going to focus much on the actual _design_ of responsive websites, but the techniques you can use to implement responsiveness on websites.
 
-The word 'design' refers to making decisions about what kinds of things look nice, crafting a user experience, and deciding exactly what a project should look like. There might be some tidbits about that sprinkled throughout these lessons, but it is not the primary focus. _However_, since this phrase has been pretty much universally adopted, it's the phrase we'll use here.
+"Responsive Design" is the term used to describe creating websites that *respond* to changes in browser size in order to create something that works on any device. Unfortunately, that phrase is a little bit misleading, due to the word 'design'. These lessons are not going to focus much on the actual *design* of responsive websites, but the techniques you can use to implement responsiveness on websites.
+
+The word 'design' refers to making decisions about what kinds of things look nice, crafting a user experience, and deciding exactly what a project should look like. There might be some tidbits about that sprinkled throughout these lessons, but it is not the primary focus. *However*, since this phrase has been pretty much universally adopted, it's the phrase we'll use here.
 
 ### So what is it?
- Since the release of the first iPhone in 2007, it has become a _requirement_ that your web projects function correctly on everything from normal desktop monitors to tiny phone screens.
 
-As you have likely discovered, this kind of responsiveness doesn't often happen automatically, especially once your projects get more complex than simple text on a page (your first HTML-only recipe project probably worked just fine on a phone, aside from the images getting cut off). So, when it comes down to it, <span id='what-is-responsive-design'>"Responsive Web Design" is a set of techniques you can use to make your sites work on any size screen.</span> 
+ Since the release of the first iPhone in 2007, it has become a *requirement* that your web projects function correctly on everything from normal desktop monitors to tiny phone screens.
```

Comment:
```suggestion
Since the release of the first iPhone in 2007, it has become a *requirement* that your web projects function correctly on everything from normal desktop monitors to tiny phone screens.
```

Stray space here I think